internal/handlers/telegram/telegram.go

```diff
@@ -60,14 +62,12 @@ func (tg *Client) StartBot(errChan chan<- error) {
 		if update.Message == nil {
 			continue
 		}
-		//formatted := "<" + update.Message.From + "> " + update.Message.Text
-		//fmt.Println("Sending message:", update.Message.Text)
-		//tg.SendMessage(formatted)
+		formatted := tg.Settings.Prefix +
+			update.Message.From.UserName +
+			tg.Settings.Suffix +
+			update.Message.Text
```

Comment:
Might be cleaner to use a format string here, especially since we already have the prefix and suffix at client creation time. Not going to block on this, but it's worth cleaning up imo